not-to-release/sources/email/enronsent33_01.xml.conllu

```diff
@@ -2155,12 +2155,12 @@
 5	the	the	DET	DT	Definite=Def|PronType=Art	6	det	6:det	_
 6	meeting	meeting	NOUN	NN	Number=Sing	3	nmod	3:nmod:of	_
 7	will	will	AUX	MD	VerbForm=Fin	13	aux	13:aux	_
-8	be	be	AUX	VB	VerbForm=Inf	13	cop	13:cop	_
+8	be	be	AUX	VB	VerbForm=Inf	0	root	0:root	_
 9	on	on	SCONJ	IN	_	13	mark	13:mark	_
 10	how	how	SCONJ	WRB	PronType=Int	13	mark	13:mark	_
 11	we	we	PRON	PRP	Case=Nom|Number=Plur|Person=1|PronType=Prs	13	nsubj	13:nsubj	_
 12	should	should	AUX	MD	VerbForm=Fin	13	aux	13:aux	_
-13	build	build	VERB	VB	VerbForm=Inf	0	root	0:root	_
+13	build	build	VERB	VB	VerbForm=Inf	8	ccomp	8:ccomp	_
```

Comment:
Another thing to watch out for when changing the root is punctuation. The end-of-sentence punctuation should attach to the root.